tests/test_blocks.py

```diff
@@ -153,9 +170,25 @@ def test_custom_page_streamfield_default_blocks():
     assert content.count("block-image") == 2
 
 
+@pytest.mark.django_db
+def test_page_chooser_block():
+    value = wagtail_factories.PageChooserBlockFactory()
+    page = Page.objects.last()
+
+    assert value == page
+
+
 @pytest.mark.django_db
 def test_image_chooser_block():
     value = wagtail_factories.ImageChooserBlockFactory()
     image = Image.objects.last()
 
     assert value == image
+
+
+@pytest.mark.django_db
+def test_document_chooser_block():
+    value = wagtail_factories.DocumentChooserBlockFactory()
+    document = Document.objects.last()
+
+    assert value == document
```

Comment:
One thing that would be good to address in these tests is whether or not the chooser block factories respect the [build strategy](https://factoryboy.readthedocs.io/en/stable/introduction.html#strategies) provided by the user. Can we add explicit tests for `.build` and `.create`, and test that the subfactory object is only persisted to the DB when the strategy is `create`?

Reply:
Thanks @jams2. I've added tests for the strategies.